### DISCUSSION TITLE
Authentication via Python

### DISCUSSION THREAD

**Discussion by rafaelkappaundb** [supabase/supabase#7493] [Questions]
Hi guys, just new here, but I'm having a problem when performing a test connecting Python with Supabase.
I installed the sybase-client package
I made the call below

from supabase_client import Client 
from dotenv import dotenv_values

config = dotenv_values(".env")

supabase = Client( 
 api_url=config.get("SUPABASE_URL"),
 api_key=config.get("SUPABASE_KEY)
) 

**ERROR : 
Exception: Supabase API-URL  is required**

But I'm correctly putting the key data and url provided by supabase.

![image](https://user-images.githubusercontent.com/108304755/176515622-bf33cb08-02ab-40ed-addb-f98197ef6bb3.png)

**Comment by burggraf** [comment]
Can you verify the value stored in `config.get("SUPABASE_URL")` before you call the `Client` function?

**Comment by JabriHassane** [comment]
Hey,
try with : 
```
from decouple import config
from supabase import create_client
SUPA_URL = config('SUPABASE_URL')
SUPA_KEY = config('SUPABASE_KEY')
supabase = create_client(SUPA_URL, SUPA_KEY)
```
